checker/raw_result.go

```diff
@@ -97,12 +110,43 @@ type Issue struct {
 	// TODO: add fields, e.g., state=[opened|closed]
 }
 
+// Commit represents a commit.
+type Commit struct {
+	// Note: SHA is not directly accessible from a pull request.
+	// TODO:SHA          string
+	Committer User
+	// Note: not all commits are reviewed.
+	// `nil` indicate no reviews.
+	Review *Review
+}
+
 // MergeRequest represents a merge request.
 type MergeRequest struct {
-	URL string
+	Number int
+	Author User
 	// TODO: add fields, e.g., State=["merged"|"closed"]
 }
 
+// User represent a user.
+type User struct {
+	Login string
+}
+
+// ReviewPlatform represents a review platform.
+type ReviewPlatform struct {
```

Comment:
If as of now there are no other fields, maybe just use a string instead of a struct. We can create one later when needed.

Reply:
that's true. Bit if we need to add, we'll always break backward compatibility. I see what you mean: it feels overkill. But would it be nice to not have backward compatibility, especially if users have to write policies?

Reply:
this is no longer exposed as we've decided to use repo-interface raw results.